main.go

```diff
@@ -55,6 +57,7 @@ func main() {
 	var tempChartStoreTargetDir string
 	var disableRemediation bool
 	var cleanupDownloadedCharts bool
+	var mode string
```

Comment:
Can we make this a `bool` since there are no other mode options other than `debug`.
In case you want to keep it as a string updated the flag description from `"Enable debug mode for the appgroup controller"` to `"Enable debug mode for the appgroup controller. Options ("", "debug")"`

Reply:
Yeah, unfortunately `debug` is already a value for `golang` and not overridable, this was the first implementation but switched over to this, maybe something like `--local`